qiskit/optimization/problems/objective.py

```diff
@@ -0,0 +1,529 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+import copy
+import numbers
+from collections.abc import Sequence
+from logging import getLogger
+from typing import Callable, List
+
+from cplex import SparsePair
+
+from qiskit.optimization.utils import BaseInterface, QiskitOptimizationError
+
+CPX_MAX = -1
```

Comment:
All `CPX_` constants have been removed and we're using enums for the different objects instead. E.g. in `constraints.py` we have a `ConstraintSense` enum instead of previously a `CPX_` constant.